src/hatti/views/table.cljs

```diff
@@ -95,7 +95,17 @@
        :rowHeight 24
        :enableTextSelectionOnCells true})
 
-(defn sg-init [data form is-filtered-dataview?]
+(defn bind-external-slick-grid-event-handlers
```

Comment:
Unfortunately no, could test at a higher level by adding a handler, initiating the event the handler is attached to, and checking that the handler is triggered as expect